browser/main/Main.js

```diff
@@ -144,6 +152,28 @@ class Main extends React.Component {
     }
   }
 
+  handleFullScreenButton (e) {
+    this.setState({ fullScreen: !this.state.fullScreen }, () => {
+      const noteDetail = document.querySelector('.NoteDetail')
+      const mainBody = document.querySelector('#main-body')
+      const sliderRight = document.querySelector('#slider-right')
+      const sliderLeft = document.querySelector('#slider-left')
+      if (this.state.fullScreen) {
+        this.state.noteDetailWidth = noteDetail.style.left
+        this.state.mainBodyWidth = mainBody.style.left
+        noteDetail.style.left = '0px'
+        mainBody.style.left = '0px'
+        sliderRight.style.display = 'none'
+        sliderLeft.style.display = 'none'
+      } else {
+        noteDetail.style.left = this.state.noteDetailWidth
+        mainBody.style.left = this.state.mainBodyWidth
+        sliderRight.style.display = 'block'
+        sliderLeft.style.display = 'block'
+      }
+    })
+  }
```

Comment:
If you scan your mouse left to right very slowly in the edit mode. You should be able to find the vertical sliders that allow you to move the lists left and right. That is if you turned off the slider code. 

![sample](https://cloud.githubusercontent.com/assets/1581329/26613369/01c109b2-4580-11e7-9165-7b2b88dfa85c.gif)

I think those names are better, columns make more sense.